cmd/podman/machine/init.go

```diff
@@ -118,6 +124,8 @@ func initMachine(cmd *cobra.Command, args []string) error {
 		vm  machine.VM
 	)
 
+	initOpts.NoInfo = initOpts.Quiet || initOpts.NoInfo
```

Comment:
where are initOpts.Quiet or initOpts.NoInfo set, they are never assigned anywhere I can see. The flags use startOpts which seems wrong.